__tests__/app.test.js

```diff
@@ -36,3 +37,18 @@ describe("ANY /notapath", () => {
       });
   });
 });
+
+describe("GET /api", () => {
+  test("200: should respond with an object describing all the available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body).toBe("object");
+        const { endpoint } = body;
+        expect(endpoint).toEqual(endpoints);
+        expect(endpoint).toHaveProperty("GET /api");
+        expect(endpoint["GET /api"]).toHaveProperty("description");
```

Comment:
This all looks good! However, you could simplify the test more by just expecting the response to equal the endpoints.json file if you want to do this 🙂